From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Wed, 15 Aug 2018 23:27:26 +0200
Subject: [PATCH] {PRINTF} sound/soc/intel/haswell: do not prepend signature
 with 0x

signature looks like $SST, no need for 0x...

Found with printk format checker plugin
---
 sound/soc/intel/haswell/sst-haswell-dsp.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/intel/haswell/sst-haswell-dsp.c b/sound/soc/intel/haswell/sst-haswell-dsp.c
index 88c3f63bded9..945982a1ee87 100644
--- a/sound/soc/intel/haswell/sst-haswell-dsp.c
+++ b/sound/soc/intel/haswell/sst-haswell-dsp.c
@@ -97,7 +97,7 @@ static int hsw_parse_module(struct sst_dsp *dsp, struct sst_fw *fw,
 	    type != SST_HSW_MODULE_LPAL)
 		return 0;
 
-	dev_dbg(dsp->dev, "new module sign 0x%s size 0x%x blocks 0x%x type 0x%x\n",
+	dev_dbg(dsp->dev, "new module sign %s size 0x%x blocks 0x%x type 0x%x\n",
 		module->signature, module->mod_size,
 		module->blocks, type);
 	dev_dbg(dsp->dev, " entrypoint 0x%x\n", entry_point);
-- 
